float64 sonar1
float64 sonar2
float64 sonar3
float64 sonar4
float64 sonar5
float64 cliff1
float64 cliff2
